### ISSUE TITLE
Conform to documented coding conventions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sshyran/genxdm#46]
```
As per issue 45, we want to document our coding convention.

This task is to get us over the hump of initially conforming to them.
```

Original issue reported on code.google.com by `eric%tib...@gtempaccount.com` on 1 Feb 2011 at 8:49
- Blocked on: #30

**Comment by GoogleCodeExporter** [comment]
Original comment by `eric%tib...@gtempaccount.com` on 1 Feb 2011 at 8:49
- Changed state: **New**
- Added labels: **Maintainability**, **Type-Task**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Started - configured Eclipse projects to use Checkstyle plugin to catch initial 
concerns.
```

Original comment by `eric%tib...@gtempaccount.com` on 27 Apr 2011 at 12:12
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
I'm thinking that we need to take this as a process, not a goal. It can't ever 
be closed, so long as there are new checkins. So ... let's say that we've got 
it underway and have it as part of the process (or that we don't and need to 
put something else in place which means reopen until we do that), and close as 
task-completed.
```

Original comment by `aale...@gmail.com` on 26 Jul 2012 at 7:47
- Changed state: **Done**
- Now blocked on: #30
